pycaret/tests/test_time_series.py

```diff
@@ -470,6 +470,23 @@ def test_tune_model_custom_folds(load_data):
     assert len(metrics2) == custom_fold + 2  # + 2 for Mean and SD
 
 
+def test_tune_model_alternate_metric(load_data):
+    """tests model selection using non default metric"""
+    exp = TimeSeriesExperiment()
+    fh = 12
+    fold = 2
+
+    exp.setup(data=load_data, fold=fold, fh=fh, fold_strategy="sliding")
```

Comment:
it may be a good idea to later define a fixture for setup with `fold_strategy="sliding"`

Reply:
I think we will have to re-architect the unit tests at some point after alpha release. For example, we could have a fixture for create models also so that could be reused in all tests instead of creating them again in each test.